microsoft-edge/webview2/get-started/winui.md

```diff
@@ -451,6 +451,20 @@ The following classes aren't accessible in WinUI 3:
 
 * `CoreWebView2EnvironmentOptions`
 * `CoreWebView2ControllerOptions`
+<!-- todo: remove items from list? -->
+
+
+<!-- ---------- -->
+###### WinAppSDK supports custom WebView2 environments
+<!-- todo: move to better section -->
```

Comment:
Where to put this info in the docs?

Live docs: 
https://learn.microsoft.com/microsoft-edge/webview2/get-started/winui#winui-3-webview2-special-considerations
@david-risney @DmitriyKomin @krschau

Reply:
I'm not sure offhand. In the step-by-step section we only add the WebView2 in the XAML, we don't also have C# code to initialize the WebView2 which is where this information would fit best. Perhaps still at the end of Step 4 as a note perhaps would work?

Reply:
i pushed a commit to modify SmartScreen section to make it appear in PR > Files Changed, to enter PR comments in it.  @david-risney

Reply:
I guess it depends on if we want to have the extra notes about this in the relevant getting started step or have the steps more clean and have the notes at the end.

Reply:
both layouts of docs are provided now; added little jump section:
```
<!-- ====================================================================== -->
## WinUI 3 WebView2 special considerations

...

<!-- ------------------------------ -->
#### WinAppSDK support for custom WebView2 environments

See [WinAppSDK supports custom WebView2 environments](#winappsdk-support-for-custom-webview2-environments), above.
```